apidoc/lib/common.js

```diff
@@ -38,7 +38,7 @@ exports.PRETTY_PLATFORM = {
 	iphone: 'iPhone',
 	ipad: 'iPad',
 	tizen: 'Tizen',
-	windowsphone: 'Windows Phone'
+	windowsphone: 'Windows'
```

Comment:
We should also remove all `tizen` and `blackberry` references, maybe even in an own PR if too much noise here.